frontend/packages/core/src/Input/text-field.tsx

```diff
@@ -191,6 +192,7 @@ const TextField = ({
   endAdornment,
   autocompleteCallback,
   defaultValue,
+  autocompleteFullWidth = true,
```

Comment:
I would switch this to just be `fullWidth = true`

Since its a valid prop for both autocomplete and textfield, then you can just do `fullWidth` on both Autocomplete and TextField